dangerfile.js

```diff
@@ -71,3 +73,27 @@ const snapshotsModified = fileChanges.some(filePath => (
 if (componentSourceFilesModified && !snapshotsModified) {
   warn('Package source files (e.g. `package/packageName/src/packageName.js`) were updated, but snapshots weren\'t. Have you checked that the tests still pass?'); // eslint-disable-line max-len
 }
+
+// Ensure shrinkwrap changes are intentional.
+const shrinkwrapUpdated = includes(modifiedFiles, 'changelog.md');
+if (shrinkwrapUpdated) {
+  warn('`npm-shrinkwrap.json` was updated. Ensure that this was intentional.');
+}
+
+// New files should include the Backpack license heading.
+const unlicensedFiles = createdFiles.filter((filePath) => {
+  if (filePath.match(/\.(js|css|scss|sh)/)) {
+    const fileContent = fs.readFileSync(filePath);
+    return !fileContent.includes('Licensed under the Apache License, Version 2.0 (the "License")');
+  }
+  return false;
+});
+if (unlicensedFiles.length > 0) {
+  fail(`These new files are do not include the license heading: ${unlicensedFiles.join(', ')}`);
```

Comment:
what if we're adding a new file in e.g. new file in the dist folder in `bpk-svgs`?

Reply:
oh merged already! I guess we'll find out 😄

Reply:
Hmm yeah good point. Maybe it should have just been a `warn` in the first place.

Reply:
I'm adding some files in, if it's breaking something we'll fix it

Reply:
So I think we have two options here:

1) Change `fail` to `warn`.
2) Change Danger to disregard files inside `dist` folders.

Option 1 is my preference - I see it like the changelog where Danger will be right most of the time but sometimes we disregard the warning for good reason.

@matteo-hertel @matthewdavidson @georgegillams thoughts?

PS this is already merged so any changes will come in another one.

Reply:
It should be impossible to add a new file without the license header. If there are directories of files that we need to ignore then that's fine (analogous to `.gitignore` / `.eslintignore`)

Reply:
If we only have danger emitting warnings then eventually they'll become visual noise - humans will ignore them and eventually they'll cease to be of use.

Reply:
Yeah I think we should disregard anything autogenerated (which I think is just inside `dist`??)

Reply:
https://github.com/Skyscanner/backpack/pull/144 sorts this out